qualtran/drawing/qpic_diagram.py

```diff
@@ -0,0 +1,332 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Classes for drawing latex diagrams for bloqs with QPIC - https://github.com/qpic/qpic.
+
+QPIC is not a dependency of Qualtran and must be manually installed by users via
+`pip install qpic`.
```

Comment:
why?

Reply:
License issues. QPIC has a [GPL-3.0 license](https://github.com/qpic/qpic#GPL-3.0-1-ov-file) license which is restricted but I confirmed with the open source team that having a dependency like this (where we don't explicit depend on the tool but can produce output the tool can consume) is fine.

Reply:
do you want to make a note that you can install it with pip but this module will invoke the qpic executable via subprocessing?